### ISSUE TITLE
Error when using loading a save in map editor on latest version of BP ext. mod

### ISSUE THREAD

**Issue by npuldon** [dewiniaid/BlueprintExtensions#15]
Error when loading my testing world in map editor on the latest version of your mod.  Hope the screenshot helps.  Love your mod otherwise.

https://i.imgur.com/StxAtRC.png

**Comment by sparr** [comment]
```
The mod Blueprint Extensions caused a non-recoverable error.
Please report this error to the mod author.

Error while running event BlueprintExtensions::on_configuration_changed
__BlueprintExtensions__/modules/landfill.lua:111: attempt to index field 'BlueprintExtensions' (a nil value)
stack traceback:
	__BlueprintExtensions__/modules/landfill.lua:111: in function '?'
	__BlueprintExtensions__/control.lua:60: in function 'call_module_methods'
	__BlueprintExtensions__/control.lua:96: in function <__BlueprintExtensions__/control.lua:93>
```

PS @npuldon you can copy and paste in the error dialog

**Comment by dewiniaid** [comment]
Should be fixed in 0.4.5, available shortly.
